### PR TITLE
Safe unserialize function

### PR DESCRIPTION
**Pull request by rodchyn**

### CODE REVIEW

File: generated/Exceptions/UnserializeException.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ * Time: 8:02 PM
+ */
+
+namespace Safe\Exceptions;
+
+
+class UnserializeException extends AbstractSafeException
+{
+
+}
```
Comment (roukmoute) [inline generated/Exceptions/UnserializeException.php]: Missing carrière return

File: generated/unserialize.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+function unserialize($str, array $options = null)
+{
+    error_clear_last();
+    $result = \unserialize($str, $options);
+    if(null === $result) {
+        throw UnserializeException::createFromPhpError();
+    }
+
+    return $result;
+}
```
Comment (roukmoute) [inline generated/unserialize.php]: Missing carrière return

File: generated/Exceptions/UnserializeException.php [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?php
+/**
```
Comment (roukmoute) [inline generated/Exceptions/UnserializeException.php]: Remove theses lines

File: generated/unserialize.php [hunk 4]
```diff
@@ -0,0 +1,65 @@
+<?php
+/**
```
Comment (roukmoute) [inline generated/unserialize.php]: Remove theses lines

**Comment by Kharhamel** [comment]
Any news on this feature?

**Pull request closed without merging**